app/controllers/api/tags_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Api::TagsController < Api::TenantController
+  def create
+    @thread = @tenant.message_threads.find(params[:message_thread_id])
+    @tag = @tenant.tags.find_by(name: tag_params)
+    raise ActiveRecord::RecordNotFound unless @tag
+
+    @thread.tags << @tag if @thread.tags.exclude?(@tag)
```

Comment:
Tu by som ocakaval, ze mozem poslat viac tagov a aj ich to vyrobi ak neexistuju.

Reply:
OK, spravil som tak, ako som pochopil. Len takto to teda nefunguje ako klasicky POST/create, ale ako custom akcia, co to v skutocnosti je. To nam nevadi?